app/model/editions/templates/TemplateHelpers.scala

```diff
@@ -39,3 +40,58 @@ object TemplateHelpers {
 
 }
 
+case object EditionType extends Enumeration() {
+
+  type EditionType = Value
+  val Regional, Special, Training = Value
+
+  implicit val readsEditionType: Reads[templates.EditionType.Value] = Reads.enumNameReads(EditionType)
+  implicit val writesEditionType: Writes[templates.EditionType.Value] = Writes.enumNameWrites
+}
+
+case class Header (title: String, subTitle: Option[String] = None)
+object Header {
+  implicit val readsHeader: Reads[Header] = Json.reads[Header]
+  implicit val writesHeader: Writes[Header] = Json.writes[Header]
+}
+
+trait EditionDefinition {
+  val title: String
+  val subTitle: String
+  val edition: String
```

Comment:
Is there a possibility to make this an enum for a stricter type?

Reply:
I don't think so: the string is defined by the template, rather than the template having to match a list stored somewhere else.